### PR TITLE
Neat report

### PR DESCRIPTION
**Pull request by manycoding**
Partially covers #119 :
1. Mark rules SKIPPED if corresponding tags were not found
2. Simplify job errors
3. Partially switched to Markdown to fix non escaped html entities in garbage symbols 
Before:
Screenshot 2019-06-25 at 16 19 34
After:
Screenshot 2019-06-25 at 18 02 55

### CODE REVIEW

File: src/arche/report.py [hunk 1]
```diff
@@ -58,12 +58,15 @@ def write_rule_outcome(cls, outcome: str, level: Level = Level.INFO) -> None:
     def write_details(self, short: bool = False, keys_limit: int = 10) -> None:
         for result in self.results.values():
             if result.detailed_messages_count:
-                self.write_rule_name(
-                    f"{result.name} ({result.detailed_messages_count} message(s))"
+                display(
```
Comment (ejulio) [inline src/arche/report.py]: If `display` and `Markdown` are "always" together, maybe a custom function could be helpful

File: src/arche/rules/duplicates.py [hunk 2]
```diff
@@ -12,28 +12,24 @@ def check_items(df: pd.DataFrame, tagged_fields: TaggedFields) -> Result:
...
-            f"{len(duplicates)} duplicate(s) with same name and url", errors=errors
-        )
+        result.add_info(Outcome.SKIPPED)
+        return result
+    result.items_count = len(df)
+    errors = {}
+    name_field = name_fields[0]
+    url_field = url_fields[0]
+    df = df[[name_field, url_field]]
+    duplicates = df[df[[name_field, url_field]].duplicated(keep=False)]
```
Comment (ejulio) [inline src/arche/rules/duplicates.py]: Shouldn't it be `df[df.duplicated(keep=False)]`?

File: src/arche/tools/helpers.py [hunk 3]
```diff
@@ -36,7 +36,7 @@ def is_job_key(value):
 def ms_to_time(ms):
     if ms is None:
         return None
-    return str(d.timedelta(milliseconds=ms))
+    return str(d.timedelta(milliseconds=ms))[:-7] or str(d.timedelta(milliseconds=ms))
```
Comment (ejulio) [inline src/arche/tools/helpers.py]: `:-7` looks really magic here

File: src/arche/report.py (hunk 1 above)
Comment (manycoding) [reply src/arche/report.py]: I don't see much benefits, the readability is good.
I wanted to think about it once the report is shaped more or less.

File: src/arche/tools/helpers.py (hunk 3 above)
Comment (manycoding) [reply src/arche/tools/helpers.py]: So this looked like the most elegant way to deal with 0 ms :)
I just want to remove those ms at all
```
str(d.timedelta(milliseconds=1000))

'0:00:01'

str(d.timedelta(milliseconds=1111))

'0:00:01.111000'

str(d.timedelta(milliseconds=12345678901))

'142 days, 21:21:18.901000'
```

Comment (ejulio) [reply src/arche/tools/helpers.py]: So, maybe placing it in a function stating the behavior.
Otherwise it looks really magic :stuck_out_tongue:

Comment (manycoding) [reply src/arche/tools/helpers.py]: The case when it doesn't work is only when the last 3 digits is 000 and I think it'll never be the case. So I removed or.

File: src/arche/rules/duplicates.py (hunk 2 above)
Comment (manycoding) [reply src/arche/rules/duplicates.py]: I had a second look and ended up with refactor.

**Comment by manycoding** [comment]
Ok, that's it. Now duplicates look like this:
Screenshot 2019-06-28 at 15 52 54
Screenshot 2019-06-28 at 15 52 47

**Pull request merged**